src/Conversion/ONNXToTOSA/ONNXToTOSALegalizeUtils.hpp

```diff
@@ -0,0 +1,91 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//==== ONNXToTosaLegalizeUtils.hpp - ONNX dialects to TOSA lowering Utils-===//
+//
+// Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+// Copyright (c) 2022 Advanced Micro Devices, Inc.
```

Comment:
I'm not suggesting we already change if there aren't any questions, but Luis recently pointed me to an example (https://github.com/amd/UIF/blob/main/samples/tf_sample.py) which we could implement here as something like:
```
// Copyright 2020 The TensorFlow Authors. All Rights Reserved.
//
// Modifications notified per clause 4(b) of the license:
// Copyright (c) 2022 Advanced Micro Devices, Inc.
```

Reply:
So we leave it like this until someone complains?